test/check-application

```diff
@@ -43,10 +43,10 @@ class TestApplication(testlib.MachineCase):
         b.reload(ignore_cache=True)
         b.wait_present("#content")
         # menu label (from manifest) should be translated
-        b.wait_text("#host-apps a[href='/starter-kit']", "Bausatz")
+        b.wait_text("#host-apps a[href='/certificates']", "Bausatz")
 
-        b.go("/starter-kit")
-        b.enter_page("/starter-kit")
+        b.go("/certificates")
+        b.enter_page("/certificates")
         # page label (from js) should be translated
         b.wait_in_text(".container-fluid p", "Läuft auf")
```

Comment:
Have tou actually run those tests? This does not seem like correct tests anyway.

Reply:
I have not written any tests yet, I would like to have some design approval before writing tests for UI.